addons/hivedr/src/main/java/org/apache/falcon/hive/util/HiveDRStatusStore.java

```diff
@@ -49,7 +49,10 @@
     private static final Logger LOG = LoggerFactory.getLogger(DRStatusStore.class);
     private FileSystem fileSystem;
 
-    private static final String DEFAULT_STORE_PATH = BASE_DEFAULT_STORE_PATH + "hiveReplicationStatusStore/";
+    private static final String DEFAULT_STORE_PATH = StringUtils.removeEnd
+            (DRStatusStore.BASE_DEFAULT_STORE_PATH,  File.separator) + File.separator
```

Comment:
I see you try to append File.separator if it is not at the end of the string. StringUtils::appendIfMissing looks better in this case.